internal/pkg/instrumentors/runner.go

```diff
@@ -28,27 +27,24 @@ import (
 )
 
 // Run runs the event processing loop for all managed Instrumentors.
-func (m *Manager) Run(ctx context.Context, target *process.TargetDetails) error {
+func (m *Manager) Run(target *process.TargetDetails) error {
```

Comment:
Removing the `ctx` arg means this cannot be canceled and shut down appropriately. I don't think that is an appropriate change, right?